### PR TITLE
add registration test WINNF.FT.S.REG.14

### PR DESCRIPTION
**Pull request by janetliuboston**

### CODE REVIEW

**Comment by janetliuboston** [comment]
The commit name is wrong. It is WINNF.FT.S.REG.14

File: src/harness/testcases/registration_testcase.py [hunk 1]
```diff
@@ -360,6 +360,67 @@ def test_WINNF_FT_S_REG_12(self):
...
+    # Device #2 is Category A with one conditional parameter missing
+    self.assertTrue(device_c['cbsdCategory'], 'A')
+    del device_c['installationParam']['indoorDeployment']
+
+    # Device #3 is Category B
+    self.assertTrue(device_b['cbsdCategory'], 'B')
+
+    # Device #4 is Category B with one conditional missing and conditionals pre-loaded
+    self.assertTrue(device_d['cbsdCategory'], 'B')
+    conditionals_c = {'registrationData': [
```
Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: Shouldn't this be conditionals_d and be referencing device_d instead?

File: src/harness/testcases/registration_testcase.py [hunk 2]
```diff
@@ -360,6 +360,67 @@ def test_WINNF_FT_S_REG_12(self):
...
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)
+    # Check registration response
+    self.assertTrue('cbsdId' in response['registrationResponse'][0])
+    for resp in response['registrationResponse']:
+        self.assertFalse('measReportConfig' in resp)
+    self.assertEqual(response['registrationResponse'][0]['response']['responseCode'], 0)
+    self.assertEqual(response['registrationResponse'][1]['response']['responseCode'], 200)
+    self.assertEqual(response['registrationResponse'][2]['response']['responseCode'], 200)
+
```
Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: I think you also want to check that the final response also has response code 200.

**Comment by janetliuboston** [comment]
Fixes made. Thanks!

**Pull request merged**